### PR TITLE
Add a modal to cancel order

### PR DESCRIPTION
**Pull request by tnagorra**
- Only dismount Page when order is placed
- Only support MODERATOR, SCHOOL_ADMIN and PUBLISHER

Closes https://github.com/kitab-bazar/kitab-bazar/issues/37
Closes https://github.com/kitab-bazar/kitab-bazar/issues/116

### CODE REVIEW

File: app/Base/context/NavbarContext.ts [hunk 1]
```diff
@@ -11,6 +13,10 @@ export const NavbarContext = createContext<NavbarContextInterface>({
         // eslint-disable-next-line no-console
         console.error('setNavbarVisibility called on NavbarContext without a provider', value);
     },
+    reset: () => {
+        // eslint-disable-next-line no-console
+        console.error('trying to reset view');
```
Comment (frozenhelium) [inline app/Base/context/NavbarContext.ts]: Please provide appropriate message
```suggestion
        console.error('reset called on NavbarContext without a provider');
```

File: app/views/Register/RegisterForm/SchoolForm/index.tsx [hunk 2]
```diff
@@ -97,10 +97,10 @@ function SchoolForm<K extends string>(props: Props<K>) {
...
             {/*
             <TextInput
-                name="vatNumber"
-                label={strings.vatNumberInputLabel}
-                value={value?.vatNumber}
-                error={error?.vatNumber}
+                name="schoolId"
+                label={strings.schoolIdLabel}
+                value={value?.schoolId}
+                error={error?.schoolId}
```
Comment (AdityaKhatri) [inline app/views/Register/RegisterForm/SchoolForm/index.tsx:103]: Why this change?

**Review by AdityaKhatri — APPROVED** [review]

**Review by frozenhelium — APPROVED** [review]
